apps/microsoft-teams/frontend/src/components/config/SearchableList/SearchableList.tsx

```diff
@@ -13,7 +14,13 @@ interface Props<T> {
  * implementation of the search options, as well as the rendering of filtered items is left up to
  * the consumer of this component.
  */
-const SearchableList = <T,>({ list, searchQuery, searchKeys, renderListItem }: Props<T>) => {
+const SearchableList = <T,>({
+  list,
```

Comment:
I wonder if at this stage it's a worth a rename of this prop to `items`? I feel like it'd be a lot more clear and also make explicit the connection between items and pinned items. And also "render list item" 😁 

Purely a subjective naming suggestion, totally optional if you think it makes sense. (But it would open a can of worms to rename elsewhere in the implementation)

Reply:
Hadn't thought about that, but definitely makes sense.  I'll update the name, it feels correct to have all 3 props named in the same ballpark.

- [ ] rename `list` to `items`